docs/conf/prague/2020/about.rst

```diff
@@ -8,7 +8,7 @@ We invite you to join hundreds of other folks for a three-day event to explore t
 The Write the Docs conference covers any topic related to documentation in the software industry.
 Past talks have also covered such diverse topics as empathy, the history of math symbols, and using emoji to keep your users' attention.
 
-Write the Docs brings *everyone* who writes the docs together in the same room: Writers, developers, support engineers, community managers, developer relations, and more!
+Write the Docs brings *everyone* who writes the docs together in the same room: Writers, Developers, Developer Relations, Customer Support, and more.
```

Comment:
Not sure why this was changed, the old version looks fine and actually is more detailed...

Reply:
I'm not sure either. This change is from Portland 2020.

Reply:
Sounds like a mistake. Can we add Editors in there as well?

Reply:
Made a suggestion to change it back :)